airbase/pom.xml

```diff
@@ -51,7 +51,7 @@
     <properties>
         <!-- see http://docs.codehaus.org/display/MAVENUSER/POM+Element+for+Source+File+Encoding -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.build.targetJdk>8</project.build.targetJdk>
+        <project.build.targetJdk>17</project.build.targetJdk>
```

Comment:
Why 17?

Reply:
Preparation for maven 4.0.0

Reply:
I guess what I’m asking is why not 21?

Reply:
I've dropped this commit. We can revisit it later.